lib/bolt/cli/src/commands/admin/mod.rs

```diff
@@ -10,12 +10,23 @@ pub enum SubCommand {
 		#[clap(subcommand)]
 		command: team_dev::SubCommand,
 	},
+	Login {
```

Comment:
Needs a 1-sentence docs description that gets auto-generated for `bolt admin --help`

Reply:
https://github.com/rivet-gg/rivet/pull/177/commits/c27b91dda64f2d0a6d02e30f9aa36e526c740193